packages/slice-machine/src/components/List/List.css.ts

```diff
@@ -50,8 +50,10 @@ export const header = style([
   }),
   {
     selectors: {
-      [`${windowStyles.tabsContent} > ${root} > &:not(:first-child)`]: {
-        marginTop: vars.space[16],
+      ["&:not(:first-child)"]: {
+        borderTopStyle: "solid",
```

Comment:
Slice Machine UI doesn't use `ListItem` currently. Should we remove the component, which would simplify `List`'s implementation?

For reference, the solid border was to prevent this UI where the new empty state is shown above a `ListHeader`:

<img width="532" alt="image" src="https://github.com/prismicio/slice-machine/assets/8601064/17a3431c-0687-4df9-994c-89f669abd176">

Note the dark gray to light gray transition, which looks odd in my opinion.